main.js

```diff
@@ -1,24 +1,84 @@
+// Need to optionally shim `crypto.getRandomValues` and esbuild needs the
+// import to come before importing `wasm_exec.js`
+import "./polyfills/crypto.js";
+
 // @ts-check
-import { readFileSync } from "fs";
-import { join } from "path";
-import { initSync, __wasm } from "./module.js";
+/* global Go */
+import "./wasm_exec.js";
+import mainWasm from "./main.wasm";
```

Comment:
Now that we are inlining the WASM, and doing 2 builds, we don't need the `module.js` file anymore.  Both cjs and esm can be built from `main.js`